util/pingpong.c

```diff
@@ -323,6 +323,9 @@ void pp_banner_options(struct ct_pingpong *ct)
 	PP_DEBUG("  - %-20s: %s\n", "iterations", iter_msg);
 	PP_DEBUG("  - %-20s: %s\n", "provider",
 		 ct->hints->fabric_attr->prov_name);
```

Comment:
Hmm we should probably also add a `NULL` check around the provider name and any other strings around here. It seems that the `printf` implementation on my systems seems to print `(null)` if given a `NULL` pointer, but I think it's technically undefined behavior. 

Not sure if that belongs in this PR or a follow-up PR, but just thought of it because of your addition.